Documentation/ceph-upgrade.md

```diff
@@ -234,50 +235,20 @@ time without compatibility support and without prior notice.
 
 Let's get started!
 
-### 1. Update modified permissions
```

Comment:
wow, there aren't any rbac changes for 1.2?

Reply:
I don't think we've had any.

Reply:
Yeah that would be ideal. For now we could diff the yamls between releases. Manual upgrade testing should also reveal if there are issues.